code/modules/customitems/item_defines.dm

```diff
@@ -426,6 +426,71 @@
 
 //////////// Masks ////////////
 
+obj/item/clothing/mask/gas/fluff/tyhailer // T0EPIC4U: Ty Omaha
+	name = "tacticool security gas mask"
+	desc = "A tacticool Security gas mask with modified 'Compli-o-nator 3000' device, it's labeled: \"Ty Omaha\". It looks like it has some..residue..you probably don't want to touch it."
+	icon_state = "sechailer"
+	var/phrase = 1
+	actions_types = list(/datum/action/item_action/halt, /datum/action/item_action/adjust)
+
+/obj/item/clothing/mask/gas/fluff/tyhailer/ui_action_click(mob/user, actiontype)
+	if(actiontype == /datum/action/item_action/halt)
+		tyhalt()
+	else
+		switch(phrase)
+			if(1)
+				to_chat(user, "\blue You set the special snowflake restrictor to: Stop or Bash")
```

Comment:
On all of these, please don't use color macros, use span classes.
`to_chat(user, "<span class='notice'>You set the special snowflake restrictor to: Stop or Bash</span>")`